### PR TITLE
OCD Fixes

### PR DESCRIPTION
**Pull request by IronCretin**
Fixed a few part descriptions.

### CODE REVIEW

File: Gamedata/Bluedog_DB/Parts/Saturn/bluedog_Saturn_S4B_SLA.cfg [hunk 1]
```diff
@@ -31,7 +31,7 @@ category = Payload
 subcategory = 0
 title = Sarnus-SIVB-LAM Lander Adapter Module
 manufacturer = Bluedog Design Bureau
-description = This adapter opens in four 'petals' to separate the Kane-11 Service Engine's hidden attach node from a Sarnus stack. Staging the decoupler opens the petals and releases the CSM. The Sina MEM is attached to the payload structure on the inside. Manually decouple the MEM once the transposition and docking manuever is complete.
+description = This 3.35m adapter opens in four 'petals' to separate the Kane-11 Service Engine's hidden attach node from a Sarnus stack. Staging the decoupler opens the petals and releases the CSM. The Sina MEM is attached to the payload structure on the inside. Manually decouple the MEM once the transposition and docking manuever is complete.
```
Comment (jsolson) [inline Gamedata/Bluedog_DB/Parts/Saturn/bluedog_Saturn_S4B_SLA.cfg]: I'm going to merge, just a note for later, 3.35m is wrong. It's a 3.75m to 2.5m adapter. Great job!

Comment (jsolson) [inline Gamedata/Bluedog_DB/Parts/Saturn/bluedog_Saturn_S4B_SLA.cfg]: Can't for the life of me figure out how to resolve the merge conflict.

Conflicting files
Gamedata/Bluedog_DB/Parts/Saturn/bluedog_Saturn_S2_Interstage.cfg

I think it's conflicting with your previous pr oddly.

**Comment by jmrd98** [comment]
This one was nasty. I'm submitting a new PR with the changes applied properly. The patches in this PR were all applied haphazardly to the development stream of the last few days, so all sorts of patches were applying to things that didn't exist. 

@IronCretin, the patches are very appreciated, but it is very helpful if you make sure to rebase/squash your commits onto the current upstream master before submitting them for PR.

**Comment by jsolson** [comment]
Sorted out the conflict locally.

**Pull request closed without merging**